microprofile/WFLY-12228_upgrade_microprofile-health_2.0.1.adoc

```diff
@@ -57,6 +57,7 @@ The microprofile-health-smallrye will provide its own "reporter" that aggreated
 == Non Requirements
 
 * The integration of the health HTTP endpoints or operations in EAP or WildFly probes for Kubernetes is out of scope of this feature. It will be tracked in a separate https://issues.jboss.org/browse/CLOUD-3326[CLOUD-3326] issue.
+* Health checks in subdeployments of EAR deployment is not supported.
```

Comment:
Wrong file?

Reply:
yes (I moved this non-requirement to the correct file)